tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/payments/stars/TransactionPartner.kt

```diff
@@ -48,6 +48,8 @@ sealed interface TransactionPartner {
     data class User(
         @SerialName(userField)
         val user: PreviewUser,
+        @SerialName(transactionTypeField)
+        val transactionType: String,
```

Comment:
It is better to make sealed interface `TransactionType` with describing of all possible types and `Unknown` as variant of future possible updates in Bots API

Reply:
Ok, let it be enum, will change in future if it will need be